src/lib/utils/constants.js

```diff
@@ -12,7 +16,7 @@ export const TABLE_ADDRESSES = [
   "0xf39f9671906d8630812f9d9863bBEf5D523c84Ab",
   "0x39cf1BD5f15fb22eC3D9Ff86b0727aFc203427cc",
   "0x8cE2Dee54bB9921a2AE0A63dBb2DF8eD88B91dD9",
-  "0x99519AcB025a0e0d44c3875A4BbF03af65933627",
+  "0x99519AcB025a0e0d44c3875A4BbF03af65933627"
```

Comment:
Not sure if we have linting rules. So whether we end arrays with a comma or not? If we had a linting guide, we could just get it to enforce these rules

Reply:
We do have prettier built into the project. Run this and it'll make a few changes:
@best-coder-NA 
```
npm run format
```

Reply:
I'd suggest leaving comments beside the TABLE_ADDRESSES too with the token name 😄

Reply:
Updated formatting and added comments!